web/src/components/tasks/business-formation/review/ReviewStep.test.tsx

```diff
@@ -569,4 +569,180 @@ describe("Formation - ReviewStep", () => {
+          )
+        )
+      ).toBeInTheDocument();
+      expect(screen.getByText(boardMembersQualificationsRight)).toBeInTheDocument();
+    });
+
+    it("in the by laws are specified and is displayed in the Choosing trustees in the Provisions section", async () => {
+      await renderStep(
+        { legalStructureId },
+        { hasNonprofitBoardMembers: true, nonprofitTrusteesMethodSpecified: "IN_BYLAWS" }
+      );
+      const getByMarkup = withMarkup(screen.getByText);
+      expect(
+        getByMarkup(
+          markdownToText(
+            `${Config.formation.fields.isVeteranNonprofit.choosingTrusteesReviewText} ${Config.formation.fields.isVeteranNonprofit.inByLaws}`
+          )
+        )
+      ).toBeInTheDocument();
+    });
+
+    it("in the form are specified and is displayed in the Choosing trustees in the Provisions section", async () => {
+      const trusteesMethodTerms = "Trustees method terms are specified";
+      await renderStep(
+        { legalStructureId },
+        {
+          hasNonprofitBoardMembers: true,
+          nonprofitTrusteesMethodSpecified: "IN_FORM",
+          nonprofitTrusteesMethodTerms: trusteesMethodTerms,
+        }
+      );
+      const getByMarkup = withMarkup(screen.getByText);
+      expect(
+        getByMarkup(
+          markdownToText(
+            `${Config.formation.fields.isVeteranNonprofit.choosingTrusteesReviewText} ${Config.formation.fields.isVeteranNonprofit.inForm}`
+          )
+        )
+      ).toBeInTheDocument();
+      expect(screen.getByText(trusteesMethodTerms)).toBeInTheDocument();
+    });
+
+    it("in the by laws are specified and is displayed in assets in the Provisions section", async () => {
+      await renderStep(
+        { legalStructureId },
+        { hasNonprofitBoardMembers: true, nonprofitAssetDistributionSpecified: "IN_BYLAWS" }
+      );
+      const getByMarkup = withMarkup(screen.getByText);
+      expect(
+        getByMarkup(
+          markdownToText(
+            `${Config.formation.fields.isVeteranNonprofit.distributingAssetsReviewText} ${Config.formation.fields.isVeteranNonprofit.inByLaws}`
+          )
+        )
+      ).toBeInTheDocument();
+    });
+
+    it("in the form are specified and is displayed in assets in the Provisions section", async () => {
+      const assetTerms = "Asset terms are specified";
+      await renderStep(
+        { legalStructureId },
+        {
+          hasNonprofitBoardMembers: true,
+          nonprofitAssetDistributionSpecified: "IN_FORM",
+          nonprofitAssetDistributionTerms: assetTerms,
+        }
+      );
+      const getByMarkup = withMarkup(screen.getByText);
+      expect(
+        getByMarkup(
+          markdownToText(
+            `${Config.formation.fields.isVeteranNonprofit.distributingAssetsReviewText} ${Config.formation.fields.isVeteranNonprofit.inForm}`
+          )
+        )
+      ).toBeInTheDocument();
+      expect(screen.getByText(assetTerms)).toBeInTheDocument();
+    });
```

Comment:
[sand] Since all these tests are copy-paste identical except for the field names, this is an excellent opportunity to improve readability by writing them in a loop or an `it.each` instead of writing the same three tests 4 times over.

Reply:
Will fix this afterward